ftgo-order-history-service/src/main/java/net/chrisrichardson/ftgo/cqrs/orderhistory/main/OrderHistoryServiceMain.java

```diff
@@ -1,7 +1,8 @@
 package net.chrisrichardson.ftgo.cqrs.orderhistory.main;
 
-import io.eventuate.tram.consumer.common.TramConsumerCommonConfiguration;
-import io.eventuate.tram.consumer.kafka.EventuateTramKafkaMessageConsumerConfiguration;
+import io.eventuate.tram.consumer.common.spring.TramConsumerCommonConfiguration;
+import io.eventuate.tram.consumer.kafka.spring.EventuateTramKafkaMessageConsumerConfiguration;
+import io.eventuate.tram.messaging.common.spring.TramMessagingCommonAutoConfiguration;
```

Comment:
Services should not explicitly `@Import` `TramMessagingCommonAutoConfiguration`.
That configuration should be picked up AUTOmatically but `@EnableAutoConfiguration` because of https://github.com/eventuate-tram/eventuate-tram-core/blob/master/eventuate-tram-messaging/src/main/resources/META-INF/spring.factories

Reply:
I will fix